docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,196 @@
+   ``trino``.
+
+   .. code-block:: text
+
+       helm repo add trino https://trinodb.github.io/charts
+
+#. Install the Trino helm chart. This fetches the template and uses the local
+   ``values.yaml`` file to populate the templates to generate the Kubernetes
+   configuration files.
+
+   .. code-block:: text
+
+       helm install my-trino-cluster trino/trino
+
+   You should see output as follows:
+
+   .. code-block:: text
+
+       NAME: my-trino-cluster
+       LAST DEPLOYED: Tue Sep 13 14:12:09 2022
+       NAMESPACE: default
+       STATUS: deployed
+       REVISION: 1
+       TEST SUITE: None
+       NOTES:
+       Get the application URL by running these commands:
+         export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+         echo "Visit http://127.0.0.1:8080 to use your application"
+         kubectl port-forward $POD_NAME 8080:8080
+
+   This output will differ based on your ``values.yaml`` configuration and
+   cluster name. For example, the port ``8080`` is set by the
+   ``.Values.service.port`` in the ``values.yaml``. Other values like the
+   ``.Values.image.tag`` dictate the container image tag which defaults to
+   ``latest``.
+
+#. Run ``kubectl get all`` to check that all pods, deployments, and services are
+   running properly. You should see expect to see output that shows running pods
+   and deployments and replicasets in the ready state.
+
+   .. code-block:: text
+
+       NAME                                               READY   STATUS    RESTARTS   AGE
+       pod/my-trino-cluster-coordinator-bfb74c98d-rnrxd   1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-hvl8n       1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-tcqgb       1/1     Running   0          161m
+
+       NAME                       TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)    AGE
+       service/my-trino-cluster   ClusterIP   10.96.25.35   <none>        8080/TCP   161m
+
+       NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE
+       deployment.apps/my-trino-cluster-coordinator   1/1     1            1           161m
+       deployment.apps/my-trino-cluster-worker        2/2     2            2           161m
+
+       NAME                                                     DESIRED   CURRENT   READY   AGE
+       replicaset.apps/my-trino-cluster-coordinator-bfb74c98d   1         1         1       161m
+       replicaset.apps/my-trino-cluster-worker-76f6bf54d6       2         2         2       161m
+
+Executing queries
+-----------------
+
+The pods running the Trino containers are all running on a private network
+internal to Kubernetes. In order to access them, specifically the coordinator,
+you will need to create a tunnel to the exposed service and your computer. You
```

Comment:
```suggestion
you need to create a tunnel to the exposed service and your computer. You
```

Reply:
Do we really want to recommend a tunnel? How about just setting up ingress properly including https? What do you think @nineinchnick ?

Also . this could be all in the Configuration section instead and then we can just link to it from a "Connect with the CLI section" or so .. that mostly links to the CLI docs .

And have similar ones for JDBC apps or whatever.

Reply:
For this one, let's just start with doing the tunnel method to get this merged, and we can add a more sophisticated method using Ingress types later.